packages/labs/src/components/select/multiSelect.tsx

```diff
@@ -111,7 +111,7 @@ export class MultiSelect<T> extends React.Component<IMultiSelectProps<T>, IMulti
     }
 
     private renderQueryList = (listProps: IQueryListRendererProps<T>) => {
-        const { tagInputProps = {}, popoverProps = {} } = this.props;
+        const { tagInputProps = {}, popoverProps = {}, selectedItems } = this.props;
```

Comment:
shouldn't `selectedItems = []` here be enough of a fix that you wouldn't need what you added below?

Reply:
That wouldn't be applied if `selectedItems === null`, but I guess `tagInputProps` and `popoverProps` also aren't hardened to that case. Will change.